TESTING-INSTRUCTIONS.md

```diff
@@ -3,6 +3,19 @@ Testing instructions
 
 ## Unreleased
 
+### Fix end date for last periods #6584
+1. We need to be in March 2021
```

Comment:
This may be a little tricky to test.  Maybe we could update this to:

```suggestion
1. Update your system clock to March 2021
```

Reply:
Thank you for the feedback. The changes has been pushed.